### PR TITLE
Genesis ceremony test

### PR DESCRIPTION
**Pull request by adaszko**
https://rchain.atlassian.net/browse/RCHAIN-2172

Implements https://github.com/rchain/rchain/blob/dev/docs/features.md#test-testtest_genesis_ceremonypytest_successful_genesis_ceremony

### CODE REVIEW

**Comment by adaszko** [comment]
bors try

File: integration-tests/test/rnode.py [hunk 1]
```diff
@@ -364,24 +413,31 @@ def make_bootstrap_node(
...
     cli_options: Optional[Dict] = None,
     mount_dir: Optional[str] = None,
+    wallets_file: Optional[str] = None,
 ) -> Node:
     key_file = get_absolute_path_for_mounting("bootstrap_certificate/node.key.pem", mount_dir=mount_dir)
     cert_file = get_absolute_path_for_mounting("bootstrap_certificate/node.certificate.pem", mount_dir=mount_dir)
 
     container_name = make_bootstrap_name(network)
 
+    container_command_flags = set([
```
Comment (woky) [inline integration-tests/test/rnode.py]: I feel it'd be good to use ordered set for command line flags, but AFAIK no options in rnode currently depend on order, so this is fine.

File: integration-tests/test/test_genesis_ceremony.py [hunk 2]
```diff
@@ -20,22 +28,57 @@
...
-                    assert False
+    with testing_context(command_line_options, random_generator, docker_client, bootstrap_keypair=CEREMONY_MASTER_KEYPAIR, peers_keypairs=peers_keypairs) as context:
+        with temporary_wallets_file(context.random_generator, [context.bootstrap_keypair] + context.peers_keypairs) as wallets:
+            with ready_bootstrap(context=context, cli_options=bootstrap_cli_options, wallets_file=wallets) as ceremony_master:
+                with started_peer(context=context, network=ceremony_master.network, bootstrap=ceremony_master, name='validator-a', keypair=VALIDATOR_A_KEYPAIR, wallets_file=wallets, cli_flags=peers_cli_flags, cli_options=peers_cli_options) as validator_a:
+                    with started_peer(context=context, network=ceremony_master.network, bootstrap=ceremony_master, name='validator-b', keypair=VALIDATOR_B_KEYPAIR, wallets_file=wallets, cli_flags=peers_cli_flags, cli_options=peers_cli_options) as validator_b:
+                        wait_for_block_approval(context, ceremony_master)
```
Comment (woky) [inline integration-tests/test/test_genesis_ceremony.py]: *puke*

https://stackoverflow.com/questions/893333/multiple-variables-in-python-with-statement

Comment (adaszko) [reply integration-tests/test/test_genesis_ceremony.py]: with statement can be replaced with explicit calls to `__enter__()` and `__exit__()` if I'm not mistaken, but I think deep nesting is still more readable and explicit.

**Comment by adaszko** [comment]
bors try

**Comment by adaszko** [comment]
bors try

File: integration-tests/test/test_internal.py [hunk 3]
```diff
@@ -26,10 +36,75 @@ def test_blocks_count_from_show_blocks() -> None:
...
+    assert len(output) == 1
+    block = output[0]
+    assert block['blockHash'] == '"cf42c994ff30189c35cbd007719c6bdb361b28c70ae88889a6e54b5431b8f7eb"'
+    assert block['blockSize'] == '"111761"'
+    assert block['blockNumber'] == '0'
+    assert block['version'] == '1'
+    assert block['deployCount'] == '10'
+    assert block['tupleSpaceHash'] == '"dcd6e349d5b4ca45a11811808ad7757bdfb856b093e02c7e4a2930817f179cdb"'
+    assert block['timestamp'] == '1'
+    assert block['faultTolerance'] == '-0.6666667'
```
Comment (KentShikama) [inline integration-tests/test/test_internal.py]: Hmm... this will probably need to change with my PR. Maybe just remove this?

Comment (adaszko) [reply integration-tests/test/test_internal.py]: I have all set up to at this point to do `bors r+`.  Would you mind removing that line your PR? 😇

Comment (KentShikama) [reply integration-tests/test/test_internal.py]: I would say the same thing :P

**Pull request merged**